src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionfurtherevidence/HandleEvidenceEventHandler.java

```diff
@@ -70,17 +69,17 @@ private void buildSscsDocumentFromScan(SscsCaseData sscsCaseData) {
 
         if (sscsCaseData.getScannedDocuments() != null) {
             for (ScannedDocument scannedDocument : sscsCaseData.getScannedDocuments()) {
-                if (scannedDocument != null && scannedDocument.getValue() != null
-                        && !equalsIgnoreCase(scannedDocument.getValue().getType(), COVERSHEET)) {
+                if (scannedDocument != null && scannedDocument.getValue() != null) {
                     List<SscsDocument> documents = new ArrayList<>();
                     if (sscsCaseData.getSscsDocument() != null) {
                         documents = sscsCaseData.getSscsDocument();
                     }
-                    SscsDocument sscsDocument = buildSscsDocument(sscsCaseData, scannedDocument);
-                    documents.add(sscsDocument);
-                    sscsCaseData.setSscsDocument(documents);
-                    sscsCaseData.setEvidenceHandled(workOutEvidenceHandled(sscsCaseData.getEvidenceHandled(),
-                        sscsDocument.getValue().getDocumentType()));
+                    if (!equalsIgnoreCase(scannedDocument.getValue().getType(), COVERSHEET)) {
+                        SscsDocument sscsDocument = buildSscsDocument(sscsCaseData, scannedDocument);
+                        documents.add(sscsDocument);
+                        sscsCaseData.setSscsDocument(documents);
+                    }
+                    sscsCaseData.setEvidenceHandled("Yes");
```

Comment:
Do we want to set this to Yes for the scenario when the evidence is of Other Document type?? 
this change is changing the last AC in this ticket t https://tools.hmcts.net/jira/browse/SSCS-6026

Reply:
the BA confirmed that the AC in the 6026 is too narrow and it shouldn't be like that. Therefore, this change makes sense to me